canonical/canonicalwebteam.flask-base
# chore: Bump Flask to 2.3.3 and update dependencies

## Done

- Updates Flask to 2.3.3
- Update other requirements where needed to be compatible

## QA

- I have set up demo on u.com. Check that the packages provided by Flask-Base are working:
https://ubuntu-com-14024.demos.haus/
